index.js

```diff
@@ -1,57 +1,65 @@
 /* eslint-disable no-restricted-globals */
-let listOfBooks = [];
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  static listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
 
-function addBook(title, author) {
-  listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
-  const serialNumber = listOfBooks.length + 1;
-  listOfBooks.push({ serialNumber, title, author });
-  localStorage.setItem('books', JSON.stringify(listOfBooks));
+  constructor({ title, author }) {
+    this.title = title;
+    this.author = author;
+    this.serialNumber = Book.listOfBooks.length + 1;
+    Book.listOfBooks.push(this);
+  }
 }
-// eslint-disable-next-line no-unused-vars
-function removeBook(event) {
-  const remSerial = event.target.parentElement.id;
+class Methods {
+  // constructor() {}
 
-  listOfBooks = JSON.parse(localStorage.getItem('books'));
-  // eslint-disable-next-line eqeqeq
-  listOfBooks = listOfBooks.filter((book) => book.serialNumber != remSerial);
+  static add({ title, author }) {
+    // eslint-disable-next-line no-new
+    new Book({ title, author });
+    localStorage.setItem('books', JSON.stringify(Book.listOfBooks));
+  }
 
-  localStorage.setItem('books', JSON.stringify(listOfBooks)); location.reload();
-}
+  static remove(event) {
+    const remSerial = event.target.parentElement.id;
+    // eslint-disable-next-line eqeqeq
+    Book.listOfBooks = Book.listOfBooks.filter((book) => book.serialNumber != remSerial);
```

Comment:
- [ ] Kindly don't use the static method. Static methods are that using them forfeits one of the advantages of objects. Objects are intended for data encapsulation. We can use the array inside the class as property and then adding and deleting the book from the array much easier. we can get an updated array as a return from the `add` and `remove` method or we can use the getter method.